admin_guide/metrics-sizing-guidelines.adoc

```diff
@@ -0,0 +1,164 @@
+
+
+If `persistent_storage=true` parameter is specified, then Openshift metrics will create PVC and use it
+as storage space for cassandra pod withing Openshift metrics. Important to note, that all what applies for cassandra database performances for case it
+it is installed out of Openshift metrics, also applies on cassandra when deployed as part of Openshift Metrics. For detailed instructions regarding cassandra databases
+is necessary to check official http://cassandra.apache.org/doc/latest/[cassandra documentation]
+
+
+- If `persistent_storage=false` is used and with intention for metrics data not to live longer than pod lifetime,
+then it is recommended to design openshift installation to have `/var` on separate partition ( to accommodate `/lib/origin/openshift.local.volumes/pods`)
+
+This is necessary as space under `/var/lib/origin/openshift.local.volumes/pods` will be used for pods by default as location where to store pod data on Openshift nodes,
+and it is necessary to ensure that massive metrics data collection will not lead to filling up storage space on openshift node and causing openshift node to be unresponsive.
+This issue should be solved when issue https://github.com/kubernetes/kubernetes/pull/27199[Initial support for pod eviction based on disk usage] is closed
+
+At this point, we assume that metrics pods are up and running after starting metrics as showed in below example
+====
+----
+ # oc get pods -n openshift-infra
+ NAME                                          READY             STATUS      RESTARTS          AGE
+ hawkular-cassandra-1-l5y4g                    1/1               Running     0                 17h
+ hawkular-metrics-1t9so                        1/1               Running     0                 17h
+ heapster-febru                                1/1               Running     0                 17h
+----
+====
+
+
+if this is not the case, please check Openshift metrics documentation and ensure Openshift metrics pods are up and running.
+
+== Openshift Metrics Dimensioning For Openshift Enterprise
+
+Openshift metrics uses Cassandra database as datastore for metrics and current version of Cassandra builtin in
+ Openshift metrics is `CASANDRA_VERSION=2.2.4` with `MAX_HEAP_SIZE=512M` and `NEW_HEAP_SIZE=100M`. It is assumed that these values should
+ cover most of Openshift metrics installations. It is possible to change these values in
+ https://github.com/openshift/origin-metrics/blob/master/cassandra/Dockerfile[cassandra docker file]
+ prior starting Openshift metrics
+
+
+Default time for Openshift metrics data preservation is specified with parameter `METRIC_DURATION`  in
+https://github.com/openshift/origin-metrics/blob/master/metrics.yaml[metrics.yaml configuration file] and has default value of 7 (days).
+This means Openshift metrics will keep data for 7 days before it start to purge old ones from cassandra database and to free
```

Comment:
When the METRICS_DURATION is reached, Cassandra will begin purging older data.  It is important to note that if a project is deleted, it's metrics remain until they are automatically aged out.